### ISSUE TITLE
WI and upper MI coordinate issues

### ISSUE THREAD

**Issue by paciorek** [PalEON-Project/PLS_products#19]
@jpeters7 I noticed the following (which is relevant for when you produce the v1.1 MI/WI/MN files):

1) that 50-100 points in WI have values in the X_COORD/Y_COORD fields that differ from the coordinates in the shape file (from just looking at coordinates(x) where 'x' is from running readOGR on the shapeFile. 

2) In upper Michigan, the X_COORD and Y_COORD values have some systematic issues compared to the coordinates in the shape file. 

Note I'm not sure if there are any issues with assigning/not assigning a proj4string to the result of the readOGR...

I guess the main thing is that when you create the CSVs you do the right thing in terms of projecting to the Paleon Albers projection, but the presence of perhaps incorrect columns is concerning.

**Comment by paciorek** [comment]
Related to this, @jpeters7 When I read the shapefiles into R, the proj4string with MN is different than WI/NoMI.

```
> attributes(mich)$proj4string
CRS arguments:
 +proj=aea +lat_1=42.122774 +lat_2=49.01518 +lat_0=45.568977
+lon_0=-83.248627 +x_0=1000000 +y_0=1000000 +datum=NAD83 +units=m
+no_defs +ellps=GRS80 +towgs84=0,0,0 
> attributes(minn)$proj4string
CRS arguments:
 +proj=utm +zone=15 +datum=NAD83 +units=m +no_defs +ellps=GRS80
+towgs84=0,0,0 
> attributes(mich)$proj4string
CRS arguments:
 +proj=aea +lat_1=42.122774 +lat_2=49.01518 +lat_0=45.568977
+lon_0=-83.248627 +x_0=1000000 +y_0=1000000 +datum=NAD83 +units=m
+no_defs +ellps=GRS80 +towgs84=0,0,0 
```

**Comment by jpeters7** [comment]
The projections were different for the original shapefiles, but there was code that converted all three to the same projection.  I tried to get the correct xy coordinates for the Great Lakes St Lawrence Albers projection in R, but couldn't.  
So I ended up reprojecting the data in GIS and then adding the xy coordinates.  I mapped all three states data along with Indiana and Illinois to make sure they all lined up. 
I have uploaded the data for Minnesota, Wisconsin and Michigan to the [wiki](https://paleon.geography.wisc.edu/doku.php/data_and_products;settlement_vegetation) (minnesota_v1.1.csv, wisconsin_v1.1.csv, and michigan_1.1.csv.)  Use the x_alb and y_alb coordinates for all three files.
